.travis.yml

```diff
@@ -8,4 +8,4 @@ jdk:
 before_script:
   - sudo chmod +x /usr/local/bin/sbt
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION jvm/test
+  - sbt "++${TRAVIS_SCALA_VERSION}!" jvm/test
```

Comment:
What does `!` mean in this case?

Reply:
force set scalaVersion. sbt 1.x changed `++` semantics. see https://github.com/sbt/sbt/pull/2613 https://github.com/sbt/website/issues/620

Reply:
Thanks!